src/esthri/Cargo.toml

```diff
@@ -84,8 +84,8 @@ version = "0.3"
 optional = true
 
 [dependencies.tokio]
-version = "1.5"
-features = [ "io-std", "net", "rt", "time",]
+version = "1.6"
+features = [ "io-std", "net", "rt", "time", "rt-multi-thread"]
```

Comment:
Maybe the `"rt-multi-thread"` feature should be optional?

Reply:
I think ideally the rt and rt-multi-threaded features should be optional/only enabled for the cli. I don’t think esthri as a library needs a run time, just the AsyncWrite/AsyncRead traits

Reply:
@notoriaga looks like we need the "rt" feature at least since I'm using `tokio::task::spawn_blocking`

Reply:
Ahh I see, because the etag stuff is blocking? Probably could be replaced with tokio fs but what you're doing is pretty much how that works afaik